src/ReferenceRepository.php

```diff
@@ -35,7 +36,7 @@ class ReferenceRepository
      * List of named references to the fixture objects
      * gathered during fixure loading
      *
-     * @phpstan-var array<class-string, array<string, object>>
+     * @phpstan-var array<class-string, array<string|int, object>>
```

Comment:
I have updated this to reflect the reality you described, which lead me to more static analysis fixes.